src_erl/Communication_Layer/http_Nerlserver/src/source/sourceStatem.erl

```diff
@@ -192,7 +195,7 @@ Triplets =getHostPort(WorkersNames,WorkersMap,NerlnetGraph,MyName,[]),
 sendSamples(ListOfSamples,_CSVPath,_ChunkSize,_LengthOfSample, Ms,Pid,_Triplets,Counter,NumOfBatchesToSend,_Method) when NumOfBatchesToSend=<0->
   receive
   after Ms ->
-    gen_statem:cast(Pid,{finishedCasting,Counter,ListOfSamples}), io:format("sent all samples~n")
+    gen_statem:cast(Pid,{finishedCasting,Counter,ListOfSamples}), io:format("sent all samples~n")   %, should exit(normal)?
```

Comment:
What is the meaning of comment here?

Reply:
Small comment I made to myself, changed to describe actual flow